src/AlgebraicGeometry/Schemes/SpecOpen/Morphisms/Methods.jl

```diff
@@ -20,7 +20,7 @@ function Base.show(io::IO, ::MIME"text/plain", f::SpecOpenMor)
   kY = length(str)
   push!(co_str, str)
   k = max(length.(co_str)...)
-  println(io, "Morphism")
+  println(io, "Spec open morphism")
```

Comment:
Not for this PR, but for later discussion:
`Spec open` is not a standard wording. Maybe it should be printed as a typename or with a different wording.

Reply:
Yes I agree. To be honnest I was not sure what to do with this without having very long names. At least the printing looks consistent and it carries the wording of the typename (for now)